neofetch

```diff
@@ -820,7 +820,7 @@ image_source="auto"
 #       Slackware, SunOS, LinuxLite, OpenSUSE, Raspbian,
 #       postmarketOS, and Void have a smaller logo variant.
 #       Use '{distro name}_small' to use the small variants.
-ascii_distro="auto"
+ascii_distro="EshanizedOS"
```

Comment:
Please don't change the default ascii_distro in the default config. If you change this to EshanizedOS, all distros will show up as EshanizedOS, which is not good.

If you just want to test it, you can use the `--ascii_distro` argument